### ISSUE TITLE
Unable to drag items using arrow keys (JAWS)

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/energy-skate-park#563]
**Test device**
Asus Zenbook

**Operating System**
Win 11

**Browser**
Chrome + JAWS

**Problem description**
From https://github.com/phetsims/qa/issues/1381

When selecting items such as the skater, stopwatch, the measuring tape, the Reference height or the red points of the track, they cannot be dragged using the arrow keys (using A,D,W,S seems to work). This only happens when the screen reader is on.

**Steps to Reproduce**
1. Go to the Intro screen.
2. Using tab navigation, add the stopwatch to the play area.
3.  Try to move the item using the arrow keys. Initially, it moves in small steps, but if you continue pressing the keys, the position stops updating.

**Visuals**

https://github.com/user-attachments/assets/b7774728-eb3c-47f5-ad9b-3131016db5fe

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Energy Skate Park‬
URL: https://phet-dev.colorado.edu/html/energy-skate-park/1.4.0-dev.5/phet/energy-skate-park_all_phet.html
Version: 1.4.0-dev.5 2026-04-06 17:36:49 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/146.0.0.0 Safari/537.36
Language: en-US
Window: 1440x731
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}
<details/>

**Comment by samreid** [comment]
SkaterNode, for instance, is just using `const keyboardDragListener = new SoundKeyboardDragListener( {`.  @jessegreenberg any ideas how to investigate this?

**Comment by jessegreenberg** [comment]
Yes, I suspect it is because of `dragSpeed`:

(core-description-quickstart-guide.md)
> KeyboardDragListener is the preferred way to make a draggable object keyboard-accessible. Whenever possible, use
`dragDelta` (one key press -> one discrete move) instead of `dragSpeed` (move while the key is held), because many
screen-reader/OS combinations do not recognize press-and-hold interactions.

It is up to you and @arouinfar for whether you want to do this. `dragDelta` will work better with JAWS, but is less smooth and game-like.  We have sometimes opted for smooth motion in the past, see https://github.com/phetsims/circuit-construction-kit-common/issues/1245 and https://github.com/phetsims/molecule-polarity/issues/322#issuecomment-3923441060, but we sacrifice arrow keys with JAWS.

**Comment by arouinfar** [comment]
The most useful keyboard interactions for the grabbed skater would be to attach them to the track (J+T) and then move to the track start (Home). 

I confirmed with @jessegreenberg that JAWS users can still move the skater with WASD keys with `dragSpeed`, so my preference would be to leave things as-is. Closing.
